Segment.xcodeproj/project.pbxproj

```diff
@@ -461,6 +468,7 @@
 				TargetAttributes = {
 					EADEB85A1DECD080005322DA = {
 						CreatedOnToolsVersion = 8.1;
+						LastSwiftMigration = 1200;
```

Comment:
Please remove since we haven't migrated to swift yet

Reply:
👍